etc/lamassu/application.properties

```diff
@@ -53,6 +53,14 @@ org.entur.lamassu.baseUrl=${LAMASSU_BASE_URL}
 org.entur.lamassu.feedupdateinterval=${LAMASSU_FEED_UPATE_INTERVAL_IN_MS:21600000}
 # How long a feed should be cached, in case it is not updated in between, in seconds, default 6h
 org.entur.lamassu.feedCacheMinimumTtl=${LAMASSU_FEED_CACHE_MINIMUM_INTERVAL_IN_S:21600}
+# For entities, we set the cache (min and max ttl) to the feed cache interval as well.
+# Note that the cache ttl should be greater than 
```

Comment:
Can you add an explanation *why* we do this, to make sure that future maintainers (us or others) don't run into the same problem as you did recently?

Reply:
Done with [10c0c36](https://github.com/mobidata-bw/ipl-orchestration/pull/244/commits/10c0c36c503d4325a93736c08d59ae3cebf92cde)